ccmlib/scylla_node.py

```diff
@@ -38,6 +39,52 @@
 if TYPE_CHECKING:
     from ccmlib.scylla_cluster import ScyllaCluster
 
+class ScyllaKeyType:
```

Comment:
Note that `scylla types` works with regular types -- i.e. values belonging to regular columns -- not just keys. So this name is misleading.

Reply:
Ack. I misunderstood the usage. I have the updated the code now.